src/Command/PullRequest/PullRequestMergeCommand.php

```diff
@@ -208,16 +194,20 @@ protected function execute(InputInterface $input, OutputInterface $output)
                 $this->addClosedPullRequestNote($pr, $mergeCommit, $squash, $input->getOption('switch'));
             }
 
+            if ($pr['user'] !== $this->getParameter($input, 'authentication')['username']) {
+                if ($patComment = $this->givePatToPullRequestAuthor($pr, $input->getOption('pat'))) {
```

Comment:
I left this separated because I'd like to extend this check giving the option to delete the remote branch if the PR's author is the authenticated user.